x-pack/plugins/security_solution/public/timelines/components/flyout/pane/index.tsx

```diff
@@ -60,7 +60,7 @@ const FlyoutPaneComponent: React.FC<FlyoutPaneComponentProps> = ({
         onClose={handleClose}
         size="100%"
         ownFocus={false}
-        style={{ visibility: visible ? 'visible' : 'hidden' }}
+        style={{ display: visible ? 'block' : 'none' }}
```

Comment:
nit: That would be nice somehow to keep the info for the future dev, that we are using the defaultElement `div` for the `EuiFlyout`, and the styles like `display: block` is the right default state.

Reply:
I will go back and leave a comment after 8.2, hopefully the test descriptions explain in the meantime.